umap/templates/umap/map_init.html

```diff
@@ -1,7 +1,9 @@
 {% load umap_tags %}
 <div id="map"></div>
 <!-- djlint:off -->
-<script type="text/javascript">
-  let MAP = new L.U.Map("map", {{ map_settings|notag|safe }})
+<script defer type="text/javascript">
+  window.addEventListener('DOMContentLoaded', (event) => {
+    let MAP = new L.U.Map("map", {{ map_settings|notag|safe }});
```

Comment:
What is the scope of this?
Why `let`?
Does it become a global variable?

Reply:
This is a (previously introduced) hack to have access to MAP everywhere, which was broken with these changes. I've updated accordingly.